--- mt-st-1.1/mtio.h.orig	2020-08-25 16:24:43.215203705 +0200
+++ mt-st-1.1/mtio.h	2020-08-25 16:24:47.328514754 +0200
@@ -136,7 +136,6 @@
 #define MTIOCVOLINFO	_IOR('m',  8, struct mtvolinfo) /* info about volume */
 #define MTIOCGETSIZE    _IOR('m',  9, struct mttapesize)/* get cartridge size*/
 #define MTIOCFTFORMAT   _IOWR('m', 10, struct mtftformat) /* format ftape */
-#define MTIOCFTCMD	_IOWR('m', 11, struct mtftcmd) /* send QIC-117 cmd */
 
 /* Generic Mag Tape (device independent) status macros for examining
  * mt_gstat -- HP-UX compatible.
